subprojects/gdk_rust/gdk_common/src/model.rs

```diff
@@ -114,7 +114,24 @@ pub struct CreateTransaction {
     pub utxos: Option<GetUnspentOutputs>,
     /// Minimum number of confirmations for coin selection
     pub num_confs: Option<u32>,
-    pub confidential_utxos_only: Option<bool>,
+    #[serde(default)]
+    pub confidential_utxos_only: bool,
+}
+
+impl Default for CreateTransaction {
```

Comment:
Why did this require changing to a manual Default implementation? Seems like it would work the same if it was derived?

Reply:
it's not needed now, removed, thanks!